draft-ietf-quic-qpack.md

```diff
@@ -217,7 +235,7 @@ contains entries which cannot be evicted.
 A dynamic table entry cannot be evicted immediately after insertion, even if it
 has never been referenced. Once the insertion of a dynamic table entry has been
 acknowledged and there are no outstanding references to the entry in
-unacknowledged header blocks, the entry becomes evictable.  Note that
+unacknowledged representations, the entry becomes evictable.  Note that
```

Comment:
representations are not acknowledged.   We lost the definition of "header block", and now use 'encoded field section' ?